Dockerfile

```diff
@@ -7,11 +7,15 @@ RUN cargo build --release && \
     cd / && rm -rf /source
```

Comment:
Why delete the source? The container gets dropped anyways no?

Reply:
Good point :slightly_smiling_face:

Reply:
On the other hand, it might still make sense to remove the source directory (which includes the `target` dir and might become quite large), because then those files will not become part of the Docker layer. The container gets dropped but the image still remains until it's removed, so I'd keep the `rm` command to save some disk space.

@dns2utf8 do you agree?

Reply:
I kind of agree. You could also run `docker image prune` afterwards.
Append the flag `--force` to get rid of the question.